src/components/CaraouselHome/index.js

```diff
@@ -0,0 +1,183 @@
+import React from "react";
+import "./Carousel.css";
+import { SLIDE_INFO } from "./../../constant/properties";
+
+import Box from "@material-ui/core/Box";
+import Fade from "@material-ui/core/Fade";
+import Paper from "@material-ui/core/Paper";
+
+// const data = [
+//   "FIRST CONTENT",
+//   "SECOND CONTENT",
+//   "LAAAAAARGE CONTENT",
+//   "MORE CONTENT",
+//   "ANITA LAVA LA TINA",
+//   "HANNAH",
+//   "LA RUTA NATURAL",
+//   "MORE SPANISH CONTENT",
+//   "EXTRA 1",
+//   "EXTRA 2",
+//   "EXTRA 3",
+//   "EXTRA 4",
+//   "EXTRA 5",
+// ];
+const dataPerPage = 3;
+
+const durationEnter = 1500;
+const durationExit = 500;
+
+class CarouselTestIndex extends React.Component {
+  constructor() {
```

Comment:
Why is this a class component when most of your components are functional with hooks?

Reply:
I have researched for custom carousel in react, So I also used class component. but I can convert it into Hooks. Will do it and create PR. Was working on the other component, that's why I missed changing.